### PR TITLE
Fix remoate oath create remote project

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes

Updated the create remote synchronized project page to use Ant Design with the working Oauth component.

## Related issue

N/A

## Checklist

Things for the developer to confirm they've done before the PR should be accepted:

*   [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
*   [x] Tests added (or description of how to test) for any new features.
*   [x] User documentation updated for UI or technical changes.

### CODE REVIEW

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]

File: src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/ProjectsController.java [hunk 1]
```diff
@@ -654,8 +569,8 @@ private DTProject createDataTablesProject(Project project) {
 	 * Response class for a {@link Project} and its {@link RemoteStatus}
 	 */
 	public class ProjectByApiResponse {
-		private RemoteStatus remoteStatus;
-		private Project project;
+		private final RemoteStatus remoteStatus;
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/ProjectsController.java:572]: That internal class is not used. Removed in cf21801406f8704b290fe2765b38dd26ac6bd17f

File: src/main/webapp/resources/js/apis/remote-api/remote-api.js [hunk 2]
```diff
@@ -1,13 +1,48 @@
...
+
+/**
+ * Delete a specific remote api
+ * @param id - identifier for a remote api
+ * @returns {Promise<AxiosResponse<any>>}
+ */
+export function deleteRemoteApi({ id }) {
+  return axios.delete(`${BASE_URL}/${id}/delete`);
+}
+
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/apis/remote-api/remote-api.js:35]: Updated in 76197dc

File: src/main/webapp/resources/js/components/remote-api/CreateRemoteProjectSyncForm.jsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+                  showSearch
+                  options={projects}
+                  placeholder={i18n("NewProjectSync.project.placeholder")}
+                />
+              </Form.Item>
+              <Form.Item
+                required
+                label={
+                  <span>
+                    REMOTE PROJECT URL
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/components/remote-api/CreateRemoteProjectSyncForm.jsx]: Updated in 822fbe4

File: src/main/webapp/resources/js/components/remote-api/CreateRemoteProjectSyncForm.jsx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+                    <Checkbox onChange={(e) => setManual(e.target.checked)}>
+                      Set url manually
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/components/remote-api/CreateRemoteProjectSyncForm.jsx]: Updated in 822fbe4

File: src/main/webapp/resources/js/pages/admin/components/remote-connections/RemoteApiStatus.jsx [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  function updateRemoteApi(event) {
+    if (
+      event.origin === window.location.origin &&
+      event.data === "remote_api_connect"
+    ) {
+      checkApiStatus();
+    }
+  }
+
+  function updateConnectionStatus() {
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/admin/components/remote-connections/RemoteApiStatus.jsx:53]: Updated in 611706c

**Review by deepsidhu85 — APPROVED** [review]

**Pull request merged**